### PR TITLE
Update job configurations

### PR DESCRIPTION
**Pull request by rakshithakamath94**
JJB (Jenkins job builder) configuration files for all the jobs that are hosted on https://ci.centos.org/view/NFS-Ganesha/

There are few changes made to support the new openshift namespace's platform,

 [1] The python scripts are converted to py2 to py3.
 [2] A small work-around has been done to get rid of curl.
 [3] Added github actions for updating the jenkins in the github push event

### CODE REVIEW

**Comment by rakshithakamath94** [comment]
Hi @kalebskeithley @ffilz ,

Please help me in getting this PR reviewed.

Thank you.

**Review by kalebskeithley — APPROVED** [review]
I don't know jenkins to be able to comment.

I trust you though. :-) I approve merging these when you're ready.

File: build_scripts/pynfs/client.py [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+print "pynfs 4.0 test output:"
+print "------------------"
+cmd = "cat %s" % log_file40
+p = subprocess.Popen(cmd, shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+pout, perr = p.communicate()
+print pout
+
+print "running pynfs 4.1"
+log_file41 = "/tmp/pynfs" + str(int(time.time())) + ".log"
+cmd = "cd /root/pynfs/nfs4.0 && ./testserver.py %s:%s --verbose --maketree --showomit --rundeps all ganesha %s > %s" %(server, export, test_parameters, log_file41)
```
Comment (dang) [inline build_scripts/pynfs/client.py]: This should be nfs4.1, not nfs4.0 (it's running the same tests as the previous run)

**Review by dang — COMMENTED** [review]
Couple of nits.  First one is likely a bug in the source scripts, but we should fix it here.  Second one is:

jobs/.macros.yml.swp should not be added

File: build_scripts/pynfs/client.py (hunk 1 above)
Comment (rakshithakamath94) [reply build_scripts/pynfs/client.py]: I have pretty much used the scripts as it is. I will make this change!

**Comment by rakshithakamath94** [comment]
I will go ahead and merge this PR.

File: build_scripts/pynfs/client.py (hunk 1 above)
Comment (dang) [reply build_scripts/pynfs/client.py]: Yeah, I assumed it was in the original script.

**Review by dang — APPROVED** [review]

**Pull request merged**